core/src/main/scala/kafka/api/ControlledShutdownRequest.scala

```diff
@@ -37,9 +37,9 @@ object ControlledShutdownRequest extends Logging {
   }
 }
 
-case class ControlledShutdownRequest(val versionId: Short,
-                                     val correlationId: Int,
-                                     val brokerId: Int)
+case class ControlledShutdownRequest(versionId: Short,
```

Comment:
These class of changes are good. I am guessing that val in a case class is redundant?